packages/trader/src/Modules/Trading/Components/Form/Purchase/cancel-deal-info.jsx

```diff
@@ -20,6 +20,7 @@ const CancelDealInfo = ({ currency, has_cancellation, proposal_info }) => {
                                 amount={cancellation.ask_price}
                                 className='trade-container__price-info-currency'
                                 currency={currency}
+                                show_currency
```

Comment:
What's the reason for not making show_currency true by default?

Reply:
Because most of the times we need it to be `false`. It's `true` just in about 6 files.